src/components/dashboard/Dashboard.js

```diff
@@ -233,15 +233,14 @@ const Dashboard = props => {
   //as they come in, currently on each new item it simply reset the feed
   //currently it seems too complicated to make it its own effect as it both depends on "feeds" and changes them
   //which would lead to many unwanted subscribe/unsubscribe to gun
-  const subscribeToFeed = () => {
-    return new Promise((res, rej) => {
-      userStorage.feed.get('byid').on(async data => {
+  const subscribeToFeed = () =>
+    getFeedPage(true).then(
```

Comment:
this is a bug on master
then should receive a function...

Reply:
fixed